src/com/SerializeMesh.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <vector>
+
+namespace precice {
+namespace mesh {
+class Mesh;
+} // namespace mesh
+namespace com {
+class Communication;
+
+namespace serialize {
+
+class SerializedMesh {
+public:
+  static SerializedMesh serialize(const mesh::Mesh &mesh);
+  void                  addToMesh(mesh::Mesh &mesh) const;
+
+  void assertValid() const;
+
+  void                  sendTo(Communication &communication, int rankReceiver);
+  static SerializedMesh receiveFrom(Communication &communication, int rankSender);
```

Comment:
```suggestion
  void                  send(Communication &communication, int rankReceiver);
  static SerializedMesh receive(Communication &communication, int rankSender);
```
To be consistent with names we use in `com`?

Reply:
IMHO, there is no additional information in the "from/to". From MPI, we are used to plain send / receive.